cypress/pages/saucedemo/inventory.js

```diff
@@ -0,0 +1,76 @@
+class inventory {
```

Comment:
Nombre de clases de JS van con capital_sentence
```suggestion
class Inventory {
```